src/financials/financialsController.ts

```diff
@@ -0,0 +1,32 @@
+import type { NextFunction, Request, Response } from 'express';
+
+import { cbordService } from '../services/cbord.service.js';
+
+export const getFinancials = async (
+  req: Request<unknown, unknown, { sessionId?: string }>,
+  res: Response,
+  next: NextFunction,
+) => {
+  try {
+    const userId = req.user?.userId;
+    if (!userId) {
+      throw new Error('Authenticated user not found');
+    }
```

Comment:
Change
`const userId = req.user?.userId;`
to
`const { userId } = req.user!;`
and delete the if statement checking if it exists after it, as if a request was able to get to this point, the auth middleware would have attacked the userId to it and req.user.userId would 100% exist